lib/controllers/BaseController.js

```diff
@@ -99,6 +103,15 @@ class BaseController {
       .throw(new ContinueWithNext());
   }
 
+  validateRequest(req, res) {
```

Comment:
@sauravmndl As discussed earlier, can you also add the check for mandatory space_guid and organization_guid in here?
Something like this.

```
    if (!_.get(req.body, 'space_guid') || !_.get(req.body, 'organization_guid')) {
      res.status(CONST.HTTP_STATUS_CODE.BAD_REQUEST).send({
        error: 'BadRequest',
        description: 'This request is missing mandatory organization guid and space guid.'
      });
    }
```

Reply:
Done